ui/src/tool-panel/location-track/location-track-infobox.tsx

```diff
@@ -400,7 +400,15 @@ const LocationTrackInfobox: React.FC<LocationTrackInfoboxProps> = ({
                                                 variant={ButtonVariant.SECONDARY}
                                                 size={ButtonSize.SMALL}
                                                 qa-id="modify-start-or-end"
+                                                title={
+                                                    splittingState
+                                                        ? t(
+                                                              'tool-panel.location-track.splitting-blocks-geometry-changes',
```

Comment:
Näihin lokalisaatioihin olisi kyllä kiva keksiä joku systeemi ettei ne levähtäisi näin pahasti täällä sisennyksissä. Yksi workaround on tietty ottaa tuo  html:n yläpuolella muuttujaan tyyliin `const splitStateTitle = splittingState ? ...`

Reply:
useTranslationille voi antaa prefixin eli esim. `const { t } = useTranslation(undefined, { keyPrefix: 'tool-panel.location-track' }`

Reply:
Sanoisin itse että tähän(kin) oikeampi ratkaisu olisi parempi komponenttijako, sillä ainakin juuri tässä nimenomaisessa tapauksessa tuo rivittyminen johtuu siitä, että indenttiä on jo lähtökohtaisesti aivan liikaa. Prefixeistä en itse kanssa kauheasti välitä, juurikin noiden Markuksen mainitsemien juttujen takia. Navigoin itsekin koodissa todella paljon käännösavainten perusteella.